sdk/CMakeLists.txt

```diff
@@ -127,10 +131,12 @@ add_library(${PROJECT_NAME} SHARED
     $<$<BOOL:${ON_TARGET}>:${ONE_CAMERA_HEADERS}>
     $<$<NOT:$<BOOL:${ON_TARGET}>>:${ALL_CAMERAS_SOURCES}>
     $<$<NOT:$<BOOL:${ON_TARGET}>>:${ALL_CAMERAS_HEADERS}>
-    $<$<BOOL:${WITH_NETWORK}>:${NETWORK_SOURCES}>
-    $<$<BOOL:${WITH_NETWORK}>:${NETWORK_HEADERS}>
-    $<$<BOOL:${WITH_NETWORK}>:${PROTO_SRCS}>
-    $<$<BOOL:${WITH_NETWORK}>:${PROTO_HDRS}>
+    $<$<NOT:$<BOOL:${WITH_JS}>>:
```

Comment:
Is this really needed? I mean if WITH_JS is ON, then WITH_NETWORK is explicitly set to OFF.